### PR TITLE
Improve colonies

### PR DESCRIPTION
**Pull request by vincentneko**
This one is long overdue !

Replace text with colonies images when needed

Screenshot 2020-09-18 18 26 31

### CODE REVIEW

**Review by alrusdi — APPROVED** [review]

**Comment by nwai90** [comment]
@vincentneko Do note that the previous text for SP "Build colony (17 MC)" and "Poseidon first action - Select where to build colony" has been replaced:

Screenshot 2020-09-19 at 9 17 08 AM

Screenshot 2020-09-19 at 9 15 37 AM

**Comment by vincentneko** [comment]
> @vincentneko Do note that the previous text for SP "Build colony (17 MC)" and "Poseidon first action - Select where to build colony" has been replaced:
> 
> Screenshot 2020-09-19 at 9 17 08 AM
> 
> Screenshot 2020-09-19 at 9 15 37 AM

Good catch ! Will update

**Comment by nwai90** [comment]
Would it make sense to do something similar for Aridor as well? Like `SelectAddColony`

Screenshot 2020-09-19 at 9 14 30 AM

**Comment by vincentneko** [comment]
> Would it make sense to do something similar for Aridor as well? Like `SelectAddColony`
> 
> Screenshot 2020-09-19 at 9 14 30 AM

Yes, I forgot this one !

File: src/Player.ts [hunk 1]
```diff
@@ -1379,7 +1379,7 @@ export class Player implements ILoadable<SerializedPlayer, Player>{
 
     private buildColony(game: Game, openColonies: Array<IColony>): PlayerInput {
 
-      let buildColony = new SelectColony("Build colony", "Build", openColonies, (colonyName: ColonyName) => {
+      let buildColony = new SelectColony("Build colony(" + constants.BUILD_COLONY_COST + " MC)", "Build", openColonies, (colonyName: ColonyName) => {
```
Comment (nwai90) [inline src/Player.ts:1385]: @vincentneko Note it's missing a space between `colony` and `(` ; should be `"Build colony (" + constants.BUILD_COLONY_COST + " MC)"` here

**Comment by vincentneko** [comment]
@nwai90 Thanks a lot for the great feedback. I've updated regarding your remarks and fixed a couple of things.
Can you please make a final review ?
CSS for the checkmark is not great but can be addressed in another PR I guess.

**Comment by nwai90** [comment]
@vincentneko Just tested, everything looks great!

**Review by nwai90 — APPROVED** [review]

**Review by pierrehilbert — APPROVED** [review]

**Pull request merged**